models/TranscribeUtterance.ts

```diff
@@ -10,9 +10,34 @@
  * Do not edit the class manually.
  */
 
-import { TranscribeUtteranceRecord } from './TranscribeUtteranceRecord';
-import { HttpFile } from '../http/http';
+import { AddToConference } from './AddToConference';
```

Comment:
I notice a lot of imports in this file that are unused. Another generator issue?

Reply:
Also seeing this in enque.ts, maybe others